.travis.yml

```diff
@@ -12,14 +12,10 @@ php:
     - 7.0
     - 7.1
     - nightly
-    - hhvm
```

Comment:
Instead of removing ci-tests on hhvm we should actually follow the advice of travis:
```HHVM is no longer supported on Ubuntu Precise. Please consider using Trusty with `dist: trusty`.```